pkg/mbedtls/include/entropy_sources_mbedtls_riot.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2020 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    pkg_mbedtls_entropy_sources Entropy source API to Mbed TLS
+ * @ingroup     pkg_mbedtls
+ *
+ * @{
+ * @file
+ * @brief       Functions to register and poll with the Mbed TLS entropy module.
+ *
+ * @author      Peter Kietzmann <peter.kietzmann@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#ifndef ENTROPY_SOURCES_MBEDTLS_RIOT_H
+#define ENTROPY_SOURCES_MBEDTLS_RIOT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Structure containing entropy function and its strength
+ */
+typedef struct {
+    mbedtls_entropy_f_source_ptr func; /**< Pointer to entropy callback func. */
+    int strong;                        /**< Strength of the entropy function
+                                         *  (strong/weak) */
```

Comment:
Is this 1 or 0? Could we use an enum?

Reply:
Yes, only 0/1 (`MBEDTLS_ENTROPY_SOURCE_WEAK`/`MBEDTLS_ENTROPY_SOURCE_STRONG`). In practice, however, this resembles mbed TLS logic. I would like to avoid translating this logic for the RIOT adaptation.

Reply:
This needs *at least* documentation on the possible values then.